src/formatDate.js

```diff
@@ -8,7 +8,61 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArr = date.split(fromFormat[3]);
```

Comment:
The code assumes that the delimiter is always at the same index in the `fromFormat` array. This assumption may not hold true if the formats are different. Consider dynamically determining the delimiter position.